docs/site/soap-calculator-tutorial-add-datasource.md

```diff
@@ -0,0 +1,139 @@
+---
+lang: en
+title: 'Add Datasource'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/soap-calculator-add-datasource.html
+---
+
+### Add a Datasource
+
+![soap-calculator-add-data-source](../../docs/img/loopback-example-soap-calculator_figure2.png)
+
+We will add a Datasource to the **LB4** application using the **CLI**. This step
+will not only create two files under the src/datasources, but also will install
+the necessary connector and its dependencies for us. Let's run the following
+command inside our application directory.
+
+```sh
+lb4 datasource calculator
+```
+
+#### Select a Connector type
+
+We specified _calculator_ as the name for our Datasource. Now **CLI** is asking
+us to select a connector type. Let's choose the **SOAP webservices** (supported
+by StrongLoop) connector from the list as shown below:
+
+```sh
+? Select the connector for calculator:
+  Oracle (supported by StrongLoop)
+  Microsoft SQL (supported by StrongLoop)
+  REST services (supported by StrongLoop)
+❯ SOAP webservices (supported by StrongLoop)
+  Couchbase (provided by community)
+  Neo4j (provided by community)
+  Kafka (provided by community)
+(Move up and down to reveal more choices)
+```
+
+#### Specify the SOAP web service endpoint
```

Comment:
github preview is not rendering this header correctly for some reason. Honestly, I don't know what the issue is

Reply:
what I did now is to re-write the same line `#### Specify the SOAP web service endpoint` in github editor and it works. I pasted it back to VsCode and apparently it is fixing that way. I inspected however and there is no special character in that line .